tests/macros/InterfaceOverview.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Test the API Interface Overview page content macro
+ *
+ * @prettier
+ */
+
+// Get necessary modules
+const fs = require('fs');
+const path = require('path');
+
+const { assert, describeMacro, beforeEachMacro, itMacro } = require('./utils');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+
+const INTERFACE_DATA = `[${JSON.stringify({
+    XMLHttpRequestEventTarget: {
+        inh: 'EventTarget',
+        impl: []
+    }
+})}]`;
```

Comment:
What's your reasoning for using JSON.stringify() here instead of just writing the string? Is that so you get syntax checking? I'm intrigued by that idea. I think that for something as short as this, I'd just write it as as string rather than bothering with  the two step thing.

If you want to keep it this way, please at least move the square brackets inside the stringify call.